pymc4/random_variables/random_variable.py

```diff
@@ -140,62 +148,21 @@ def as_tensor(self):
         if self._backend_tensor is None:
             self._backend_tensor = ctx.var_as_backend_tensor(self)
 
-        return self._backend_tensor
+        return self._bijector.forward(self._backend_tensor)
 
 
 class ContinuousRV(RandomVariable):
```

Comment:
If this and `DiscreteRV` both do nothing let's get rid of them. The extra indirection doesn't help any

Reply:
Good point.